### PR TITLE
🎉 UI updates for standardized explorer names, explore tagging, and an `explorer-tiles` component

### PR DESCRIPTION
**Pull request by ikesau**
The UI complement to https://github.com/owid/owid-content/pull/36

## Charts page:
image

## Explorer Tiles component:
image

## Explorer page:
image

## Explorer prominent link:
image

## Summary by CodeRabbit

- **New Features**
	- Added a new "Explorer Tags" management page in the Admin interface, enabling viewing, editing, and deleting of tags associated with explorers.
	- Introduced navigation to the "Explorer Tags" page from the Admin sidebar under the "Explorers" section.
	- Added functionality for displaying explorer tiles in articles, including titles, subtitles, and links to charts or data explorers.
	- Implemented a new route for fetching all explorer tags.

- **Enhancements**
	- Improved explorer page by adding support for an `explorerSubtitle`.
	- Enhanced styling for explorer download links and explorer tiles within articles.

- **Refactor**
	- Simplified admin and API router configurations by removing unnecessary code and streamlining route definitions.
	- Updated database interactions to support new explorer tags functionality.
	- Refactored Gdoc handling to support explorer tiles and improve data processing.

- **Bug Fixes**
	- Corrected logic for setting descriptions in `ProminentLink` component to handle various scenarios more effectively.

- **Database Changes**
	- Introduced a migration for creating a new `explorer_tags` table to manage explorer-tag associations.

### CODE REVIEW

**Comment by ikesau** [comment]
@coderabbitai review

File: db/model/Gdoc/GdocBase.ts [hunk 1]
```diff
@@ -554,22 +566,28 @@ export class GdocBase extends BaseEntity implements OwidGdocBaseInterface {
                         title: resolvedTitle,
                         resolvedUrl: `${BAKED_GRAPHER_URL}/${resolvedSlug}`,
                         thumbnail: `${BAKED_GRAPHER_EXPORTS_BASE_URL}/${resolvedSlug}.svg`,
+                        tags: [],
```
Comment (ikesau) [inline db/model/Gdoc/GdocBase.ts:569]: Not doing anything with chart tags atm. Can add them when we need them

**Review by mlbrgl — COMMENTED** [review]
@ikesau I'm seeing a failing test (can repro locally)

**Review by mlbrgl — APPROVED** [review]
It is so nice not to have to thread explorers throughout the codebase, well done! 🙌

Run locally, works as expected 🤝

I don't know how often this'll happen but it might be visually better to align the titles here
Screenshot 2024-02-15 at 10 35 35

Mardown making its way into prominent links (I couldn't go all the way with running `mirror_explorers` locally so I'm not sure if that would fix it so I'm raising it here) 
Screenshot 2024-02-15 at 11 21 58

File: site/gdocs/components/ExplorerTiles.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    }
+    .explorer-tiles-grid {
+        row-gap: var(--grid-gap);
+        @include md-down {
+            grid-template-rows: repeat(2, 1fr);
+            column-gap: var(--grid-gap);
+        }
+    }
+    .explorer-tile {
+        background-image: url("/explorer-thumbnail.webp");
```
Comment (ikesau) [inline site/gdocs/components/ExplorerTiles.scss:41]: Update this with new image

File: site/gdocs/components/ExplorerTiles.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    ) : null
+
+    return (
+        <a
+            className="explorer-tile span-cols-3 span-md-cols-6"
+            href={linkedChart.resolvedUrl}
+        >
+            {icon}
+            <div className="explorer-tile__text-container">
+                <p className="h3-bold explorer-tile__title">
```
Comment (ikesau) [inline site/gdocs/components/ExplorerTiles.tsx:33]: Add background blur to this

**Pull request merged**